exec/jdbc-all/pom.xml

```diff
@@ -441,7 +441,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
                   
                   </message>
-                  <maxsize>20000000</maxsize>
+                  <maxsize>21000000</maxsize>
```

Comment:
I would not worry about this too much at this time.

Laurent made a good point above. We should match the jetty-servlets version with our jetty-server version. So you might want to use 9.1.5.v20140505 instead.

[9.1.5.v20140505](http://mvnrepository.com/artifact/org.eclipse.jetty/jetty-servlets/9.1.5.v20140505)